### PR TITLE
enhanced patch to get QSC certs accepted

### PR DESCRIPTION
**Pull request by baentsch**
Addresses Issue #43 . @xvzcf Please check whether I did the patch correctly/in the same way you (wanted) it created. Hence only Draft PR.

### CODE REVIEW

**Comment by xvzcf** [comment]
Looks fine to me.

File: chromium/qsc-certs.patch [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+--- a/net/cert/internal/simple_path_builder_delegate.cc
++++ b/net/cert/internal/simple_path_builder_delegate.cc
+@@ -110,8 +110,8 @@ bool SimplePathBuilderDelegate::IsPublicKeyAcceptable(EVP_PKEY* public_key,
+     return true;
+   }
+ 
+-  // Unexpected key type.
+-  return false;
++  // Accept all keys for QSC;
++  return true;
```
Comment (dstebila) [inline chromium/qsc-certs.patch:22]: Accepting all keys seems very permissive. Is there a risk here that too much is being accepted?  Can we add a return true for the OQS algorithms but still have a return false for the remainder?

Comment (baentsch) [reply chromium/qsc-certs.patch:22]: I also thought about that -- but then had to accept I don't understand BoringSSL already good enough to facilitate such check. @xvzcf: Do you have a suggestion?

**Comment by dstebila** [comment]
@xvzcf Can this be updated now that open-quantum-safe/boringssl#28 has landed?

**Comment by baentsch** [comment]
@xvzcf Do you have an update to this PR with your BoringSSL-algorithm check or should I try to build one (rebasing everything)? Do we rebase to 0.4.0 or latest?

**Comment by xvzcf** [comment]
I do have an update, I'll put it up tomorrow or the day after and we can take it from there.

**Comment by baentsch** [comment]
Replaced by #51

**Pull request closed without merging**